Unity-Technologies/Graphics
# [9.x.x][Shader Graph] Fix Window Layout bugs

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
Backport #1298 
- Fixed a bug where the Main Preview window was no longer a square aspect ratio. [1257053](https://issuetracker.unity3d.com/product/unity/issues/guid/1257053/)
- Fixed a bug where the size of the Graph Inspector would not save properly. [1257084](https://issuetracker.unity3d.com/product/unity/issues/guid/1257084/)
- Increased the default size of the Graph Inspector to not cut off target settings. 

Broken behavior: 
![brokenWindowLayout](https://user-images.githubusercontent.com/39529353/87593162-e0182580-c69f-11ea-99d4-15040a54a777.gif)

Fixed behavior: 
![fixedWindowLayout](https://user-images.githubusercontent.com/39529353/87593180-e7d7ca00-c69f-11ea-81c2-2a49624e433b.gif)

---
### Testing status

**Manual Tests**: What did you do?
- Other: repeated manual testing on compatible versions
manually tested new graph creation and reloading existing graphs to confirm new window layouts persist. 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.
